middleware/error-handler.js

```diff
@@ -26,7 +26,9 @@ const errorHandlerMiddleware = (err, req, res, next) => {
   }
   if (err.name === 'TokenExpiredError') {
     customError.message =
-      err.userMessage || 'Session expired, please login again';
+      //err.userMessage || 'Session expired, please login again';
+      err.userMessage || 'jwt expired';
```

Comment:
Can you give a more user-friendly message to the user?  They don't know what jwt is, I bet you :)))